src/mapping-v3-core.ts

```diff
@@ -251,6 +255,30 @@ export function handleTransfer(event: Transfer): void {
   transfer.save();
 }
 
+// v3.2 core contract Enum ProjectUpdatedFields, encoded as bytes32 Bytes
```

Comment:
Nit: can you update this comment to include a permalink to the relevant lines in the contracts repo for easy reference

Reply:
good call - I wish enum values were included in the ABIs for reference, but next best thing is this - updated in [a475369](https://github.com/ArtBlocks/artblocks-subgraph/pull/289/commits/a47536973be141709ff17ddbbd662d5e335ffa26)